reporting/impl/src/main/resources/reports/templates/embedded.ftl

```diff
@@ -25,7 +25,24 @@
     </div>
     <!-- / Navbar -->
 
-    <iframe style="width:100%; height: 100%; position:absolute;" src="${reportModel.reportProperties.embeddedUrl}"></iframe>
+    <div class="container-fluid" role="main">
```

Comment:
Is this piece of code related with the change or boy-scout refactoring ?

Reply:
Yes it is.
It's the last entry (in my first comment above) in the list of changes required: 
_Tattletale report: add the standard header (with tooltip, application name and paddings) we have in all the other reports (and removing the standard border the browser adds to iframes)_